src/main/resources/common-reference.conf

```diff
@@ -5,7 +5,7 @@ logger {
 
 postgres {
   jdbcDriver = "org.postgresql.Driver"
-  url = "jdbc:postgresql://localhost/postgres"
+  url = "jdbc:postgresql://localhost:${port}/postgres"
```

Comment:
FYI not every environment has "localhost" as the host where ports are exposed.
This is why in Testcontainers we have this:
https://www.testcontainers.org/features/networking/#getting-the-container-ip-address

Reply:
oh, btw, if you would use Testcontainers, you could launch containers just by specifying a JDBC URL 😊
https://www.testcontainers.org/modules/databases/#database-containers-launched-via-jdbc-url-scheme

Reply:
@bsideup Ah, that's covered too, I just forgot to add it in the PR - https://github.com/7mind/distage-livecode/commit/d15721b6f46c7b98b1b88b73f9c0e4acca887ea6

Reply:
@bsideup Ah, that's covered too, I just forgot to add it in the PR - https://github.com/7mind/distage-livecode/commit/d15721b6f46c7b98b1b88b73f9c0e4acca887ea6